### PR TITLE
Fix: typedoc error

### PR DESCRIPTION
**Pull request by nguyentoanit**

### CODE REVIEW

**Comment by nguyentoanit** [comment]
@moltar New `typedoc` version remove `mode` option. Therefore, there will not `globals` section on the right of docs. That is bad UI. But I could not find a replacement for them.

- [Release note](https://github.com/TypeStrong/typedoc/releases/tag/v0.20.0)

File: examples/crud-negative-targeting.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import { amazonAdvertising } from './auth'
```
Comment (moltar) [inline examples/crud-negative-targeting.ts:1]: What happened with this file? Why are there these big changes in this PR? Seems unrelated?

**Comment by moltar** [comment]
> @moltar New `typedoc` version remove `mode` option. Therefore, there will not `globals` section on the right of docs. That is bad UI. But I could not find a replacement for them.
> 
>     * [Release note](https://github.com/TypeStrong/typedoc/releases/tag/v0.20.0)

Do you have an example of how it will look like?

Suggestions for fixing?

File: examples/crud-negative-targeting.ts (hunk 1 above)
Comment (nguyentoanit) [reply examples/crud-negative-targeting.ts:1]: New `typedocs` version detected errors in our `examples`.
Then, I found that I forgot to update examples when I refactor `SponsoredBrandsProductTargetingOperation` (in `68d8cd4` commit).

Previous versions didn't found them.

**Comment by nguyentoanit** [comment]
> > @moltar New `typedoc` version remove `mode` option. Therefore, there will not `globals` section on the right of docs. That is bad UI. But I could not find a replacement for them.
> > ```
> > * [Release note](https://github.com/TypeStrong/typedoc/releases/tag/v0.20.0)
> > ```
> 
> Do you have an example of how it will look like?

Do we have any hosting for containing development docs. Example: [amazon-advertising-api-sdk-test.scaleleap.org](https://amazon-advertising-api-sdk-test.scaleleap.org)

**Comment by nguyentoanit** [comment]
> > Do we have any hosting for containing development docs. Example: [amazon-advertising-api-sdk-test.scaleleap.org](https://amazon-advertising-api-sdk-test.scaleleap.org)
> 
> Not yet. But can you generate locally and just share a screenshot?

I have used entry point from `src` directory to is `index.ts` file and resolved them.
I exported `decorators`, `errors` and `gunzip` helper too.
[Here is screenshot of our docs](https://prnt.sc/wl3bmz)

**Comment by moltar** [comment]
I think that looks ok. What is your concern?

**Comment by nguyentoanit** [comment]
> I think that looks ok. What is your concern?

After i change entry point from `src` directory to is `index.ts` file, that looks better and my concern was resolved.
That screenshot is took from new docs after changing.

**Review by moltar — APPROVED** [review]
